examples/langgraph-sls-fastapi-rag/scripts/easy-options/easyoptions.sh

```
@@ -0,0 +1,212 @@
+    fi
+
+    options+=(h=help)
+    arguments=()
+
+    # Prepare known options
+    for option in "${options[@]}"; do
+        option_var=${option#*=}
+        option_name=${option%=$option_var}
+        if [[ "${#option_name}" = "1" ]]; then
+            short_options="${short_options}${option_name}"
+            if [[ "${#option_var}" > "1" ]]; then
```

Shellcheck (0.10.0)

[error] > is for string comparisons. Use -gt instead.

(SC2071)